Tests/ConfigTests/ConfigurationPropertyTests.swift

```diff
@@ -234,4 +234,36 @@ class ConfigurationPropertyTests: XCTestCase {
         let actualValue = try whenTheDeclarationIsWritten(for: imageProperty)
         expect(actualValue).to(equal(expectedValue))
     }
+
+    func testItCanWriteAnOptionalStringProperty() throws {
+        let property = ConfigurationProperty<String?>(key: "test", typeHint: "String?", dict: [
+            "defaultValue": NSNull.self
+            ])
```

Comment:
Dodgy indent here. Sorry, haven't got swiftformat set up...

Reply:
D'oh, dangers of relying on swiftformat...